### DISCUSSION TITLE
is there any way to force check repository even it's not contain a renovate.json/js

### DISCUSSION THREAD

**Discussion by joebnb** [renovatebot/renovate#32270] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

gitlab

### Please tell us more about your question or problem

is there any way to force check repository even it's not contain a renovate.json/js,i want create a global config instead repository

 INFO: Repository started (repository=hz-fe/static-shelf)
       "renovateVersion": "38.142.0"
 INFO: Repository is disabled - skipping (repository=test)
 INFO: Repository finished (repository=test)
       "cloned": true,
       "durationMs": 6269

**Comment by rarkins** [comment, marked as answer]
Yes, you can control this via the settings `onboarding` and `requireConfig`
